### DISCUSSION TITLE
Why do I get a name even though deepstack match is false ??

### DISCUSSION THREAD

**Discussion by msolomos** [skrashevich/double-take#239] [General]
Maybe is a silly question but can you please explain why even though deepstack gives no match, double-take returns a name??
Am sure I am doing something wrong, but I can't figure out what it is.

![double-take](https://github.com/skrashevich/double-take/assets/72358278/a4df0330-c909-4d03-91bc-d712123ebb17)

Here is my double-take configuration:

`
mqtt:
    host: 192.168.70.240
    port: 1883
    username: mqtt-user
    password: mosquitto
    
    
    topics:
      frigate: frigate/events      
      homeassistant: homeassistant      
      matches: double-take/matches      
      cameras: double-take/cameras  
    
    
frigate:
    url: http://192.168.70.40:5000
    attempts:
      latest: 5
      snapshot: 0
    stop_on_match: true
    min_area: 0
    
    
detectors:
    deepstack:
        url: http://192.168.70.35:5000
        timeout: 40
        

detect:
  match:
    save: true
    base64: false
    confidence: 75
    purge: 72
    min_area: 10000

  unknown:
    save: true
    base64: false
    confidence: 15
    purge: 24
    min_area: 0    
    
cameras:
  frontdoor_camera2:
    mqtt:
      bounding_box: False
      quality: 100
      height: 500`

**Comment by noelhibbard** [comment]
I'm experiencing the same thing. It will have 100% confidence and the name is correct but it still doesn't consider it a match.

**Comment by noelhibbard** [comment]
I figured it out... in my case the "box area is too low". Anything listed in red is the cause for it to not match. The confidence AND box size has to be above threshold to be a match.

The box size is determind buy multiplying the height and width and then in the config you set the threshold with this setting:

```
detect:
  match:
    min_area: 10000
```
